src/components/App/styledComponents.styles.ts

```diff
@@ -26,11 +116,25 @@ export const StyledCard = styled.section<StyledCardProps>`
   height: ${props => {
     if(props.contents === 'wedding') {
       return '40em;'
+    } else if(props.contents === 'empty') {
```

Comment:
I didn't realize you could put this level of specific conditional rendering for styled components!